js/languages/en-US.json

```diff
@@ -234,7 +234,54 @@
       "saveErrorAlertTitle": "Unable to save advanced settings"
     },
     "moderationTab": {
-      "sectionName": "Moderation"
+      "sectionName": "Moderation",
+      "disputeResolution": "Dispute Resolution",
+      "disputeResolutionText": "Sellers have the ability to select you as a Moderator for their store. If a dispute arises between them and a customer, you'll be called in to review the details of the order and determine how the order's funds should be distributed.",
+      "moderationStatus": "Moderate Disputes",
+      "pricePerDispute": "Price per dispute",
+      "pricePerDisputeOptions": {
+        "percentageOfOrder": "Percentage of Order",
+        "fixedPrice": "Fixed Fee",
+        "percentagePlusFixed": "Fixed Fee Plus Percentage"
+      },
+      "pricePerDisputeHelper": "The amount you charge to review and close a dispute",
+      "feeTypeAmount": "Price per dispute",
+      "fixed": "Fixed",
+      "percentage": "Percentage",
+      "fixedPlusPercentage": "Fixed Plus Percentage",
+      "feeTypeHelper": "The amount you charge to review and close a dispute.",
+      "description": "Description",
+      "descriptionHelper": "Add your description (e.g. What is your strategy for reviewing cases? What is your expected response time? What is your schedule of operation? During what hours are you available each day? etc.)",
+      "terms": "Terms",
+      "termsHelper": "Add your terms of service (e.g. Do you have legal limitations? What happens after you make a decisions? etc.)",
+      "languages": "Languages",
+      "languagesHelper": "Enter all of the languages you support (ie: you can communicate clearly in).",
+      "acceptGuidelines": "I have read the recommended %{acceptGuidelinesLink}",
+      "acceptGuidelinesLink": "OpenBazaar Dispute Resolution Guidelines",
+      "understandRequirements": "I understand I must offer my services to resolve disputes to anyone that requests them.",
+      "status":{
+        "saving": "Saving moderation settings...",
+        "done": "Moderation settings saved.",
+        "fail": "Saving the moderation settings failed."
+      },
+      "errors": {
+        "confirm": "To become a moderator, you must agree to both statements below.",
+        "save": "The moderation settings were not saved."
+      },
+      "moderatorModelErrors": {
```

Comment:
Can moderatorModelErrors key please be a sibling of the other model error object (e.g.'profileModelErrors')? A model can be used by multiple parts of the UI, so I don't think its errors should be nested inside a UI specific key ('moderationTab' in this case).